iree/compiler/Dialect/VM/Target/C/test/control_flow.mlir

```diff
@@ -2,17 +2,13 @@
 
```

Comment:
Looks like the cf dialect has more canonicalization patterns than the vm dialect. Probably [this one](https://github.com/llvm/llvm-project/blob/c2428a4fad3470261128fe92d79f7df8dbe6d415/mlir/lib/Dialect/ControlFlow/IR/ControlFlowOps.cpp#L238) caused the original control flow to be simplified after the conversion to the emitc dialect.